Petty/Scenes/Pet Detail/PetDetailViewController.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  PetDetailViewController.swift
+//  Petty
+//
+//  Created by chuongmd on 4/5/19.
+//  Copyright © 2019 Sun-Asterisk. All rights reserved.
+//
+
+import UIKit
+
+final class PetDetailViewController: UIViewController {
+    @IBOutlet weak var petImageView: UIView!
+    @IBOutlet weak var petDetailView: UIView!
+    @IBOutlet weak var likeNumberLabel: UILabel!
```

Comment:
thêm private cho các outlet
```suggestion
    @IBOutlet private weak var likeNumberLabel: UILabel!
```